tests/simulation/test_simulation.py

```diff
@@ -353,9 +355,15 @@ async def test_process_transaction_records(
                 puzzle_hash=await wallet_node.wallet_state_manager.main_wallet.get_new_puzzlehash(),
                 coins={coin},
             )
+            assert tx.spend_bundle is not None, "the above created transaction is missing the expected spend bundle"
             await wallet.push_transaction(tx)
 
-            await full_node_api.process_transaction_records(records=[tx])
+            if record_or_bundle == "record":
+                await full_node_api.process_transactions(records=[tx])
+            elif record_or_bundle == "bundle":
+                await full_node_api.process_transactions(bundles=[tx.spend_bundle])
```

Comment:
we should test with multiple spend bundles that have multiple additions to exercise the code in process_transactions